app/coffee/Features/Collaborators/CollaboratorsEmailHandler.coffee

```diff
@@ -2,7 +2,17 @@ Project = require("../../models/Project").Project
 EmailHandler = require("../Email/EmailHandler")
 Settings = require "settings-sharelatex"
 
-module.exports =
+
+module.exports = CollaboratorsEmailHandler =
+
+	_buildInviteUrl: (project, invite) ->
+		"#{Settings.siteUrl}/project/#{project._id}/invite/token/#{invite.token}?" + [
+			"project_name=#{encodeURIComponent(project.name)}"
+			"user_first_name=#{encodeURIComponent(project.owner_ref.first_name)}"
+			"r=#{project.owner_ref.referal_id}" # Referal
```

Comment:
Is this the referal_id for the bonus scheme? We haven't historically awarded referrals for project invites.